index.js

```diff
@@ -63,8 +63,6 @@ function getTarget (abi, runtime) {
 }
 
 var supportedTargets = [
-  {runtime: 'node', target: '0.10.48', abi: '11', lts: false},
-  {runtime: 'node', target: '0.12.17', abi: '14', lts: false},
```

Comment:
when a version is in `deprecatedTargets`, this library will find it when resolving a specific `abi`/`target`. It won't however be used for prebuild etc, as that uses `supportedTargets`